tests/dist/src/test/java/org/hawkular/apm/tests/dist/TraceCompletionTimeITest.java

```diff
@@ -405,18 +405,18 @@ public void testGetCompletionTimesThreeFragmentCausedBy() throws Exception {
 
         Trace trace1 = new Trace();
         trace1.setId("1_"+suffix);
-        trace1.setStartTime(System.currentTimeMillis() - 60000); // Within last hour
+        trace1.setStartTime(TimeUnit.MILLISECONDS.toMicros(System.currentTimeMillis()) - 4000); // Within last hour
```

Comment:
why? it does not matter, you were using it to be able to use default query (better is to use the same constant across all tests, otherwise it looks suspicious)